src/styles/index.scss

```diff
@@ -1,3 +1,48 @@
+@import './header';
+@import './main';
+
 body {
   margin: 0;
 }
+
+.main {
+  margin: 0 auto;
+  display: grid;
+
+  gap: 46px 48px;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 488px) {
+  .main {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
```

Comment:
The media query breakpoint at 488px is very close to the one at max-width: 488px. Consider using a more distinct breakpoint to avoid overlap and ensure clear transitions between styles.